sql/migrations/2023-08-23-identifier-types-sequence/up.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN TRANSACTION;
+ALTER SEQUENCE identifier_type_id_seq RESTART WITH 40;
+COMMIT;
```

Comment:
Will this work for a new user who has downloaded the dump and setup their local environment or do they have to run this migration again?

Also what if we have lets say 60 identifiers in the data dump, and then we create a new a new identifier. Will it work?

Reply:
There is an arcane incantation to use the highest ID value dynamically, see https://stackoverflow.com/questions/5342440/reset-auto-increment-counter-in-postgres/55024610#55024610

Something like this perhaps? (⚠️ untested ⚠️ )

`SELECT SETVAL('identifier_type_id_seq', (SELECT MAX(id) FROM identifier_type));`

Reply:
I did reset my db and tested this out. After applying the migration, it works well.